charts/steward/templates/podsecuritypolicy-controllers.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if or (eq .Values.runController.podSecurityPolicyName "00-steward-controllers") (eq .Values.tenantController.podSecurityPolicyName "00-steward-controllers") -}}
```

Comment:
I would not test for a "magic value".

Instead:

* In `values.yaml` set `*.podSecurityPolicyName` to null or empty string. The empty value means that the user does not provide an own PSP but the Helm chart should create one.
* In `_helpers.tpl` create some helper templates:
    * `steward.runController.podSecurityPolicyName`, `steward.tenantController.podSecurityPolicyName` and `steward.pipelineRuns.podSecurityPolicyName`: Resolve to the name of the respective PSP, based on the current values. Resolve to `00-steward-controllers` / `00-steward-run` in case the respective value is not set (null or empty).
    * `steward.controllers.generatePodSecurityPolicy` and `steward.pipelineRuns.generatePodSecurityPolicy`: Resolve to non-emtpy string (means true) if the respective PSP must be generated, or to the empty string (means false) otherwise. This is determined based on the respective values.
* Use the helper templates at the respective places.

Reply:
done in here https://github.com/SAP/stewardci-core/pull/237/commits/6f8d6f76d457de8f7df4501f78df8d516c7823f5

Reply:
See 452f7a0 for corrections and improvements.